RunLog/RunLog/Sources/Util/Extension/Int+.swift

```diff
@@ -0,0 +1,17 @@
+//
+//  Int+.swift
+//  RunLog
+//
+//  Created by 김도연 on 3/14/25.
+//
+
+import Foundation
+
+extension Int {
+    /// 3자리마다 콤마(,)를 추가하여 문자열로 변환
+    func formattedWithComma() -> String {
+        let numberFormatter = NumberFormatter()
+        numberFormatter.numberStyle = .decimal
+        return numberFormatter.string(from: NSNumber(value: self)) ?? "\(self)"
+    }
```

Comment:
```suggestion
    var formattedString: String {
      let numberFormatter = NumberFormatter()
      numberFormatter.numberStyle = .decimal
      return numberFormatter.string(from: NSNumber(value: self)) ?? "\(self)"
    }
```

넘겨야하는 파라미터가 없다면 계산속성으로 하는게 좋을 것 같은데 어떠신가요?
또한 위와 마찬가지로 형식이 있고, String이니 formattedString 네이밍은 어떠신가요

Reply:
넹 좋습니다!